rails/README.md

```diff
@@ -34,10 +34,52 @@ A guide for building great Rails apps.
 
 ## Migrations
 
+Any time there is need to change the schema or data stored in the application
+database with a given group of work, write a [strong migration].
 - Don't change a migration after it has been merged into `main` if the desired
   change can be solved with another migration.
+- Schema migrations and data migrations should be kept separate.
+- We use the `strong_migration` gem, along with a 3 phase approach for any
+  change where we are renaming or moving columns around. These phases should
+  happen over 3 different releases.
+  - Phase 1:
+    - _Schema Migration_ - Create new column in the appropriate table.
+    - _Code and/or Schema Migration_ - Write the same data to both old and new
+      places.
```

Comment:
We should mention the `fx` gem and how we write a trigger/function pair to keep the new column (follower) in sync with the old column (leader)